tools/export/iar/iar_definitions.json

```diff
@@ -157,5 +157,11 @@
     },
     "NCS36510":{
         "OGChipSelectEditMenu": "NCS36510\tONSemiconductor NCS36510"
+    },
+    "NUC472HI8AE": {
+        "OGChipSelectEditMenu": "NUC400AE series\tNuvoton NUC400AE series (NUC442AE,NUC472AE)"
```

Comment:
@bridadan Please keep the `OGChipSelectEditMenu` setting. Install [Nu-Link Driver (IAR)](http://www.nuvoton.com/hq/products/microcontrollers/arm-cortex-m4-mcus/Software/?__locale=en&resourcePage=Y) first before building the exported IAR project. The change wouldn't fix the error still per my test.